doc/symfony-cache-configuration.rst

```diff
@@ -169,6 +173,79 @@ to overwrite the method to make it public.
 
   **default**: ``null``
 
+Tagging
+~~~~~~~
+
+.. versionadded:: 2.1
+
+    Support for tag invalidation with Symfony HttpCache has been added in
+    version 2.1.
+
+To support :doc:`cache tags <response-tagging>`, register the
+``PurgeTagsListener``. The purge listener needs the special
+``Toflar\Psr6HttpCacheStore\Psr6Store`` cache store with tag support.
+
+.. note::
+
+    Symfony's ``HttpCache`` store implementation does not support tags.
+    Therefore, you need the `Toflar Psr6Store`_ which implements the Symfony
+    Store interface but supports cache tagging. See the project README for more
+    information on the store.
+
+    To install the store, run
+    ``composer require toflar/psr6-symfony-http-cache-store dev-master``.
```

Comment:
I'd remove `dev-master` here. Once we release it it's obsolete and composer will choose the right version anyway :)

Reply:
indeed. i thought we might merge this before the store is tagged, but as we have to wait for stable symfony 3.4 anyways we can wait.